editoast/src/views/infra/lines.rs

```diff
@@ -64,19 +60,22 @@ async fn get_line_bbox(
     #[expect(deprecated)]
     let infra =
         Infra::retrieve_or_fail(conn, infra_id, || InfraApiError::NotFound { infra_id }).await?;
-    let infra_cache = InfraCache::get_or_load(conn, &infra_caches, &infra).await?;
+
     let mut bbox = BoundingBox::default();
-    let mut tracksections = infra_cache
-        .track_sections()
-        .values()
-        .map(ObjectCache::unwrap_track_section)
-        .filter(|track| track.line_code == Some(line_code))
+    let selection_settings =
+        SelectionSettings::new().filter(move || TrackSectionModel::INFRA_ID.eq(infra.id));
+    let tracksections_model = TrackSectionModel::list(conn, selection_settings).await?;
+    let mut tracksections = tracksections_model
+        .into_iter()
+        .map(TrackSection::from)
+        .filter(|track| track.extensions.sncf.as_ref().unwrap().line_code == line_code)
```

Comment:
```suggestion
        .filter(|track| track.extensions.sncf.as_ref().expect("track section extension 'sncf' is required for /infra/{id}/lines/{line_code}/bbox").line_code == line_code)
```

nicer panic messages + easily grep-able string